### PR TITLE
Fix for Issue 225 - updated tests

### PR DESCRIPTION
**Pull request by shaikres**
updated integration tests for vmc_networks and vmc_dhcp_profiles execution module

### CODE REVIEW

**Review by waynew — APPROVED** [review]
I think that I'm going to approve this, to unblock the other PRs -- but the requested refactoring should still definitely happen.

File: tests/integration/modules/test_vmc_networks.py
Comment (shaikres) [reply tests/integration/modules/test_vmc_networks.py:16]: subnets is required only during create call, so i have not included it in common data dict.

network_id is not required for get networks call, so kept it in a separate fixture instead of including it in common data dict.

File: tests/integration/modules/test_vmc_networks.py
Comment (shaikres) [reply tests/integration/modules/test_vmc_networks.py:57]: sure, i will address this suggestion once all the modules got merged.

File: tests/integration/modules/test_vmc_dhcp_profiles.py
Comment (shaikres) [reply tests/integration/modules/test_vmc_dhcp_profiles.py:65]: sure, i will make this change in separate PR.

File: tests/integration/modules/test_vmc_networks.py
Comment (shaikres) [reply tests/integration/modules/test_vmc_networks.py:65]: sure, i will make this change in separate PR.

File: tests/integration/modules/test_vmc_networks.py
Comment (shaikres) [reply tests/integration/modules/test_vmc_networks.py:103]: sure, i will make this change in separate PR.

**Pull request merged**
